src/tests/conditions.test.js

```diff
@@ -479,3 +509,348 @@ describe('Conditional with a minimum value check', () => {
     expect(handleValidation({ salary: 1000, reason: 'reason_one' }).formErrors).toEqual(undefined);
   });
 });
+
+describe('Conditional with fields that depend on the previous value', () => {
+  it('Should hide dependent field if value does not satisfy conditional anymore', () => {
```

Comment:
Hi @RuslanZavacky I confess I forgot this PR and then went on vacation. In the meantime, the playground is already using the "chained conditionals" as explained above to avoid the original bug from happening.

As you didn't answer, I'm closing this PR. But if you want to cleanup the test with that explanation, go ahead, we'll merge it after :)